src/redux/store.js

```diff
@@ -1,8 +1,6 @@
 import { configureStore } from '@reduxjs/toolkit';
 import missionsReducer from './missions/missionsSlice';
-import rocketsReducer from './rockets/rocketsSlice';
-
-// import fetchRockets from './Rocket/RocketSlice';
+import rocketsReducer, { fetchRockets } from './Rocket/RocketSlice';
```

Comment:
I think that this line can be:
import rocketsReducer from './Rocket/RocketSlice';